flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionMiscITCase.java

```diff
@@ -384,4 +410,15 @@ public static class LocalDateTimeToRaw extends ScalarFunction {
             return LocalDateTime.parse(str);
         }
     }
+
+    public static byte[] serializeLocalDateTime(LocalDateTime localDateTime) {
```

Comment:
use the existing utility `org.apache.flink.util.InstantiationUtil#serializeToByteArray`

Reply:
Thanks for reviewing this @twalthr. Used `org.apache.flink.util.InstantiationUtil#serializeToByteArray` in `serializeLocalDateTime` because because it may throw `IOException`